core/dbt/task/runnable.py

```diff
@@ -360,21 +357,25 @@ def execute_nodes(self):
         pool = ThreadPool(num_threads)
         try:
             self.run_queue(pool)
-
         except FailFastError as failure:
             self._cancel_connections(pool)
+
+            node_results_ids = [r.node.unique_id for r in self.node_results]
```

Comment:
Nitty Nitty Nits: It took me a sec to understand self.node_results only contain info about nodes that got executed here. Maybe updating the name to `executed_node_ids` will make it easier to read?